uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/dropdown-button/dropdown-button.core.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-env amd */
+
+define([
+  'common/angular',
+  'common/angularBootstrap'
+], function (angular) {
+  angular.module('calendar-feeds.dropdown-button.core', []);
```

Comment:
if we don't need this module, then we can simply not have it (btw, didn't `calendar-feeds.dropdown` depend on `ui.bootstrap` before?)

Reply:
Let me check

Reply:
If it did, that was wrong anyway, it uses the Link Modal module, that **does** depend on the UIBootstrap:

![image](https://user-images.githubusercontent.com/3973243/42273191-632e663c-7f80-11e8-8ed3-4d498f513f7f.png)

But apart from the modal, the Dropdown Button component does not need `ui.bootstrap`, it uses the standard bootstrap dropdown.

Reply:
Done.